editor/plugins/game_view_plugin.cpp

```diff
@@ -861,8 +864,6 @@ void GameView::_update_arguments_for_instance(int p_idx, List<String> &r_argumen
 	N = r_arguments.insert_after(N, itos(DisplayServer::get_singleton()->window_get_native_handle(DisplayServer::WINDOW_HANDLE, get_window()->get_window_id())));
 
 #if MACOS_ENABLED
-	r_arguments.push_back("--display-driver");
-	r_arguments.push_back("embedded");
 	r_arguments.push_back("--embedded");
```

Comment:
this is only pushed `#if MACOS_ENABLED`, but is read on main on any platform, does it create any regression in other platforms?

Reply:
It shouldn't as other platforms won't use it, not is it displayed in Godot `--help`, so it is a hidden argument.

Reply:
and why make it a MacOS only argument? just asking, makes sense to use it in all platforms

Reply:
> and why make it a MacOS only argument? just asking, makes sense to use the same argumetns in all platforms

Isn't implemented for other platforms. We'll remove it from main.cpp in a subsequent PR.

Reply:
See #106166, which returns an error if specified on unsupported platforms.